data/ka'het missions.txt

```diff
@@ -135,11 +135,11 @@ mission "First Contact: Z Ka'het: Quarg 2"
 			choice
 				`	"What happened after that?"`
 			label ka'het
-			`	The Quarg stops for a few seconds before continuing, as if he was reflecting on something. "For many millennia they grew, developing brilliant pieces of technology. Then, compelled by their past experiences, they began creating weapons not only to defend themselves, but also to retaliate; these 'Ka'het', as they named them, turned on their creators shortly after they were complete.`
-			`	"For few more centuries the Builders lasted, before falling under their very own hubris. Their space is still populated by the Ka'het they gave birth to through the immense facilities they assembled, who do not yet possess a vast intelligence." After bowing, the Quarg quickly walks away, leaving you with more questions than answers.`
+			`	The Quarg stops for a few seconds before continuing, as if he was reflecting on something. "For many millennias they grew, developing brilliant pieces of technology. Then, compelled by their past experiences, they began creating weapons not only to defend themselves, but also to retaliate; these 'Ka'het', as they named them, turned on their creators shortly after they were complete.`
+			`	"For a few more centuries the Builders lasted, before falling under their very own hubris. Their space is still populated by the Ka'het they gave birth to through the immense facilities they assembled, a being that does not yet possess a vast intelligence." After bowing, the Quarg quickly walks away, leaving you with more questions than answers.`
 				decline
 	on decline
 		fail "First Contact: Ka'het: Quarg 1"
 		log "Asked the Quarg about the history of the Graveyard. Although not being told the reason for the abandoned ringworld, have learned about the previous inhabitants of the region, the Builders."
-		log "Factions" "Builders" `The Builders inhabited the Graveyard millennia ago, but have since become extinct. They developed some sort of weapon, called Ka'het, which stopped obeying their orders and rebelled against them.`
-		log "Factions" "Ka'het"  `According to the Quarg, the Ka'het were initially a weapon created by the Builders, which then rebelled and led them to extinction. Even if millennia passed, they are still being produced in the Graveyard.`
+		log "Factions" "Builders" `The Builders inhabited the Graveyard millennia ago, but have since become extinct. They developed some sort of weapon, called the 'Ka'het', which stopped obeying their orders and rebelled against them.`
```

Comment:
```suggestion
		log "Factions" "Builders" `The Builders inhabited the Graveyard millennia ago, but have since become extinct. They developed some sort of weapon, called the "Ka'het," which stopped obeying their orders and rebelled against them.`
```